css/app.css

```diff
@@ -89,19 +90,26 @@ img.blog-image + div.summary-blog-section {
 }
 
 aside.aside-note {
-    width: 40%;
     padding-left: .5rem;
-    margin-left: .5rem;
+    margin: .5rem;
     float: right;
     border-left: 3px solid #76c158;
     font-style: italic;
     color: #29627e;
 }
+@media only screen and (min-width: 768px) {
+    aside.aside-note {
+        width: 40%;
+    }
+}
 
 aside.aside-note > p {
     margin: .125rem;
 }
 
+aside.aside-note > h1, h2, h3, h4, h5, h6 {
+    font-size: larger;
```

Comment:
Does this mean that heading size in asides is ignored? I suppose that could be valid.

Reply:
Yes, I don't think different heading styles should be allowed on the aside snippet